samples/red_black_tree.cr

```diff
@@ -328,10 +328,7 @@ class RedBlackTreeRunner
   def initialize(n = 10_000)
     @n = n
 
-    random = Random.new(1234) # repeatable random seq
     @a1 = Array(Int32).new(n) { rand(99_999) }
-
-    random = Random.new(4321) # repeatable random seq
     @a2 = Array(Int32).new(n) { rand(99_999) }
```

Comment:
Shouldn't this code instead be fixed to do what it was intended to do?

Reply:
It should, but not as a part of this PR.